app/static/js/libs/marked.js

```diff
@@ -834,7 +834,7 @@ Renderer.prototype.listitem = function(text, checked) {
   }
 
   return '<li class="task-list-item">'
-    + '<input type="checkbox" class="task-list-item-checkbox"'
+    + '<input type="checkbox" class="task-list-item-checkbox" onchange = "checkUncheck(this)"'
```

Comment:
@Frederick-S Hey, just wanted to confirm. You're saying that we could create a separate custom function and call that ```renderer``` when we hit a ```list_item``` instead of overwriting the original source code, am I right?